python/orca/src/bigdl/orca/learn/pytorch/core/lifecycle.py

```diff
@@ -32,6 +32,11 @@ def setup_torch_distribute(self, tcp_store_host, tcp_store_port, world_rank,
         self.setup_components()
         self.setup_ddp_components()
 
+    def setup_torch_estimator(self, world_rank, world_size):
+        self.rank = world_rank
+        self.size = world_size
+        self.setup_components()
```

Comment:
setup components would be called twice in both `setup_torch_estimator` and `setup_torch_distribute`? This seems not reasonable, can we remove `setup_components` in `setup_torch_distribute`